scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -101,15 +103,21 @@ trait Config extends Serializable {
   def setMapSpillThreshold(count: Int): Config =
     this + (SpillableProps.MAP_THRESHOLD -> count.toString)
 
+  @deprecated("deprecated in Cascading 2.7 and dropped in Cascading 3.0, use setMapSideAggregationCapacity", "cascading 2.7")
+  def setMapSideAggregationThreshold(count: Int): Config = {
```

Comment:
This isn't really correct for scalding, the comment mentions the method is realted to the associated operations which wires down to https://github.com/twitter/scalding/blob/develop/scalding-core/src/main/scala/com/twitter/scalding/Operations.scala#L126 which uses the same key as cascading. Not sure we shouldn't just be keeping the method/key as it was just defining the constant in scalding instead